packages/compass-aggregations/src/modules/index.js

```diff
@@ -685,7 +685,6 @@ export const getPipelineFromIndexedDB = (pipelineId) => {
     fs.readFile(file, 'utf8', (error, data) => {
       if (!error) {
         const pipe = JSON.parse(data);
-        track('Aggregation Opened', { num_stages: pipe.pipeline.length });
```

Comment:
We use the same action to open the aggregation from the "my queries" screen, so moved the tracking out of here so we don't get multiple events for the same thing